agent/taskresource/envFiles/envfile.go

```diff
@@ -0,0 +1,434 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package envFiles
+
+import (
+	"sync"
+	"time"
+
+	"path/filepath"
```

Comment:
nit: you may want to group your standard library together as the first group.

Reply:
idk that's how goimports and gofmt did it.. i reordered it.